### ISSUE TITLE
error on running the docker image script 

### ISSUE THREAD

**Issue by Maryam-ghol** [FernLab/cookiecutter-py-package#14]
* {{ cookiecutter.project_name }} version: test_cookiecutter_maryam_5
* Python version: remote machine 2.7.13
* Operating System: openSUSE Leap

### Description

trying to run the docker creation script I face the below error: 
#### Build runner docker image
Sending build context to Docker daemon  4.608kB
Error response from daemon: Dockerfile parse error line 35: Unknown flag: mount
mefe4 tests/CI_docker% cat build_test_cookiecutter_maryam_5_testsuite_image.sh 

### What I Did
./build_python_boilerplate_testsuite_image.sh

**Comment by romulogoncalves** [comment]
The issue is gone when the docker version is equal or greater than 20.*. If not, the user needs to run the docker run with the following option:
```
DOCKER_BUILDKIT=1 docker build ....
```

And add the following line to the docker file (it has to be as first line, this is, even before the license SPDX lines):
```
# syntax=docker/dockerfile:experimental
```

**Comment by romulogoncalves** [comment]
Solved by #18
